airflow/sensors/bash.py

```diff
@@ -22,6 +22,7 @@
 from tempfile import NamedTemporaryFile, TemporaryDirectory, gettempdir
 from typing import Sequence
 
+from airflow import AirflowException
```

Comment:
```suggestion
from airflow.exceptions import AirflowException
```

But can we use a more specific exception class than this? AirflowException is raised in too many places, a more specific subclass can be useful for readability when you need to debug failed tasks.

Reply:
Sure - which exception did you have in mind? None of the subclasses of AirflowException look like an obvious fit.

(I'm all for readability, but I assume you don't actually want me to create an entire exception subclass to throw one exception?)

Reply:
Create a new subclass in this module and raise it. Mention the exception class (and the fact it’s an AirflowException subclass) in the hook’s docstring, so users know what to catch.

Reply:
Cool. Just to warn you, I'm terrible at naming things, so I hope you're ok with me calling it `AndrewWasHereException` :stuck_out_tongue:

Reply:
> None of the subclasses of AirflowException look like an obvious fit

Ah I lie, seems like I should have used `AirflowFailException`. I've changed to use that - if you still want me to use a dedicated sub-class let me know.

Reply:
`AirflowFailException` seems good enough. (It’s used in some weird places but that’s not your problem, this use case is appropriate.)